src/shared/component/SwaggerURLInput.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import './SwaggerURLInput.scss';
+
+class SwaggerURLInput extends React.Component {
+
+  onSubmit(e) {
+    e.preventDefault();
+
+    const swaggerURL = this.swaggerURLInput.value;
```

Comment:
`this.swaggerURLInput` is a reference to the `input` element, as documented in the [React docs](https://facebook.github.io/react/docs/refs-and-the-dom.html)